### PR TITLE
Add deviation towards the medial side

### LINKED ISSUE #832: Deviation towards the medial side of anatomical entity pattern template
1. [x] add new pattern `deviationTowardsTheMedialSideOfAnatomicalEntity.yaml` for review
2. [x] get feedback
3. [x] fix or improve pattern based on feedback

* Examples:
- [HP:0030084	Clinodactyly](https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0030084)
- [HP:0001863	Toe clinodactyly](https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0001863)
- [HP:0009486	Radial deviation of the hand](https://www.ebi.ac.uk/ols/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0009486)

### PR DESCRIPTION
**Pull request by rays22**
Add deviation towards the medial side of anatomical entity pattern template.

If applied, this commit will fix #832.

### CODE REVIEW

File: src/patterns/dosdp-dev/deviationTowardsTheMedialSideOfAnatomicalEntity.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+pattern_iri: http://purl.obolibrary.org/obo/upheno/patterns-dev/deviationTowardsTheMedialSideOfAnatomicalEntity.yaml
+
+description: 'Use this pattern for phenotypes involving angulation, curvature or bending towards the middle of an anatomical entity that is abnormal in space or time (e.g. permanent).'
+
+#  examples:
+#    - http://purl.obolibrary.org/obo/HP_0030084  # Clinodactyly
+#    - http://purl.obolibrary.org/obo/HP_0001863  # Toe clinodactyly
+#    - http://purl.obolibrary.org/obo/HP_0009486  # Radial deviation of the hand
+
```
Comment (sbello) [inline src/patterns/dosdp-dev/deviationTowardsTheMedialSideOfAnatomicalEntity.yaml:12]: http://purl.obolibrary.org/obo/MP_0006253 clinodactyly

File: src/patterns/dosdp-dev/deviationTowardsTheMedialSideOfAnatomicalEntity.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    vars:
+      - anatomical_entity
+
+  - annotationProperty: broad_synonym
+    text: "permanent bending of %s"
+    vars:
+      - anatomical_entity
+
+def:
+  text: "A changed direction of position including angulation, curvature or bending towards the middle of %s."
```
Comment (rays22) [inline src/patterns/dosdp-dev/deviationTowardsTheMedialSideOfAnatomicalEntity.yaml:55]: Avoid using curvature, curved.
Use 'shifted' or 'moved' in the definition.

File: src/patterns/dosdp-dev/deviationTowardsTheMedialSideOfAnatomicalEntity.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    - anatomical_entity
+
+annotations:
+  - annotationProperty: broad_synonym
+    text: "permanent curving of %s"
+    vars:
+      - anatomical_entity
+
+  - annotationProperty: broad_synonym
+    text: "curvature of %s"
```
Comment (rays22) [inline src/patterns/dosdp-dev/deviationTowardsTheMedialSideOfAnatomicalEntity.yaml:45]: avoid curvature

File: src/patterns/dosdp-dev/deviationTowardsTheMedialSideOfAnatomicalEntity.yaml (hunk 2 above)
Comment (rays22) [reply src/patterns/dosdp-dev/deviationTowardsTheMedialSideOfAnatomicalEntity.yaml:55]: A shift or changed direction of position towards the middle of %s.

**Comment by ybradford** [comment]
this looks good to me Orcid id 0000-0002-9900-7880

**Pull request merged**